functions/supplier/file-supplier/src/main/java/org/springframework/cloud/fn/supplier/file/FileSupplierConfiguration.java

```diff
@@ -93,10 +95,17 @@ else if (this.fileSupplierProperties.getFilenameRegex() != null) {
 	}
 
 	@Bean
-	@CustomizationAware
-	public FileInboundChannelAdapterSpec fileMessageSource(FileListFilter<File> fileListFilter) {
-		return Files.inboundAdapter(this.fileSupplierProperties.getDirectory())
+	@CustomizationDoc
```

Comment:
Why are we adding this customization only here? How come not in others? Or is it more like we just picked this one as an example and will do it for others later on?

Reply:
That's correct.
If you want me to do more, then you likely need to wait more for a final fix 😄

Reply:
The issue for enabling more components is #276

Reply:
:-) That is fine, Artem!